docs/sources/write/front-matter/index.md

```diff
@@ -68,38 +69,65 @@ The correct way to use aliases depends on whether the project is versioned or no
 
 Aliases should include a YAML comment explaining the absolute URL path that the relative path redirects.
-This helps a reviewer check that your alias works correctly.
-For example, `../path/to/alias/ # /docs/grafana/<GRAFANA VERSION>/path/to/alias/`.
+This helps a reviewer confirm that your alias works correctly.
+
+A YAML comment starts with a hash (`#`).
+
+For example, the following Markdown front matter snippet, in the file `new-url.md`, defines an alias to redirect `/docs/grafana/<GRAFANA_VERSION/original-url/`.
+The comment `# /docs/grafana/<GRAFANA_VERSION>/original-url/` indicates the absolute URL path.
+
+```
+---
+aliases:
+  - ./original-url/ # /docs/grafana/<GRAFANA_VERSION>/original-url/
+```
+
+##### Determine the relative alias
+
+To determine the relative alias, you must first understand the meaning of the current directory (`.`) and parent directory (`..`) path elements when they are used at the start of an alias.
 
-##### Examples
+The following example redirects _`SOURCE`_ to _`DESTINATION`_ using the pages:
 
-To redirect the page `/docs/grafana/latest/alerting/silences/` to `/docs/grafana/latest/alerting/manage-notifications/create-silence/`, you must add a relative alias in the source file for `/docs/grafana/latest/alerting/manage-notifications/create-silence/` containing the relative alias to `/docs/grafana/latest/alerting/silences/`.
+| Variable        | Page                                                                 |
+| --------------- | -------------------------------------------------------------------- |
+| _`SOURCE`_      | `/docs/grafana/latest/alerting/silences/`                            |
+| _`DESTINATION`_ | `/docs/grafana/latest/alerting/manage-notifications/create-silence/` |
 
-- The relative alias `./` refers to the page`/docs/grafana/latest/alerting/manage-notifications/` because that is the directory containing the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`.
-- The relative alias `../` refers to the page `/docs/grafana/latest/alerting/`.
-- The relative alias `../silences/` refers to the page `/docs/grafana/latest/alerting/silences/`.
+- The current directory element (`.`) refers to the directory containing _`DESTINATION`_.
 
-Therefore, with the alias `../silences/` in the source file for the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`, Hugo will create a redirect page at `/docs/grafana/latest/alerting/silences/`.
+  In this example, the page `/docs/grafana/latest/alerting/manage-notifications/`.
 
-To redirect the page `/docs/grafana/latest/alerting/unified-alerting/` to `/docs/grafana/latest/alerting/` you must add a relative alias in the source file for `/docs/grafana/alerting/` containing the relative alias to `/docs/grafana/latest/alerting/unified-alerting/`.
+- The parent directory element (`..`) refers to the parent directory of the current directory element.
```

Comment:
Same, this is confusing because there is no `..` in the table.   If you're giving an example, the example needs to include the `..` .

Reply:
I've completely rewritten this section again to see if that makes it clearer. 🤞